addon/components/stripe-element.js

```diff
@@ -6,7 +6,7 @@ export default Component.extend({
   classNames: ['ember-stripe-element'],
 
   autofocus: false,
-  options: {},
+  options: computed(() => ({})),
```

Comment:
Why was this change required? Ember has deprecated computed overrides recently and I'm worried that with this change, the addon will now have these deprecations.

Reply:
the change is right for the mo but as you say, isn't very future proof.

Using `this.set('options', {});` within the `init()` method might be better at the mo. Not sure when `init()` will be deprecated though.

Reply:
Using `init()` should be safe.

Reply:
Right, I wasn't aware of this. I'm now back on an Ember project after some time away, and it looks like things have changed a bit. I'm happy to do this on `init()` if that's the done thing now.

Reply:
Ah thanks! No problem, there's a number of deprecations in-flight at the mo so understandable if you've been away from Ember for a while!

Reply:
Sorted now! Let me know if this solution is up to snuff :crossed_fingers:

Reply:
Great, thanks!